src/pages/profile.tsx

```diff
@@ -79,18 +83,19 @@ const UserProfile: NextPageWithLayout = () => {
             用户姓名不能为空
           </Alert>
         )}
-        {!show && <Button
-          onClick={() => setShow(!show)}
-          fontSize='sm' variant='brand' fontWeight='500' mb='24px'>
-          修改个人信息
-        </Button>}
-
-        {show && <Button
+        {loaded && <Button
           backgroundColor='orange'
           onClick={handleSubmit}
           fontSize='sm' variant='brand' fontWeight='500' mb='24px'>
           保存
         </Button>}
+      
+        {!loaded && <Button
```

Comment:
Instead of adding a new button, use the same button to show loading state: `<Button isLoading={loading} ...>`

Reply:
Merged 2 buttons